new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/IstioUtils.java

```diff
@@ -0,0 +1,271 @@
+    ExecResult result = null;
+    StringBuffer getIngressPort = null;
+    getIngressPort = new StringBuffer("kubectl -n istio-system get service istio-ingressgateway ");
+    getIngressPort.append("-o jsonpath='{.spec.ports[?(@.name==\"http2\")].nodePort}'");
+    logger.info("getIngressPort: kubectl command {0}", new String(getIngressPort));
+    try {
+      result = exec(new String(getIngressPort), true);
+    } catch (Exception ex) {
+      logger.info("Exception in getIngressPort() {0}", ex);
+      return 0;
+    }
+    logger.info("getIngressPort: kubectl returned {0}", result.toString());
+    if (result.stdout() == null) {
+      return 0;
+    } else {
+      return new Integer(result.stdout());
+    }
+  }
+
+  /**
+   * Get the secure https ingress port of istio installation.
+   * @returns secure ingress https port for istio-ingressgateway
+  */
+  public static int getIstioSecureIngressPort() {
+    ExecResult result = null;
+    StringBuffer getSecureIngressPort = null;
+    getSecureIngressPort = new StringBuffer("kubectl -n istio-system get service istio-ingressgateway ");
+    getSecureIngressPort.append("-o jsonpath='{.spec.ports[?(@.name==\"https\")].nodePort}'");
+    logger.info("getSecureIngressPort: kubectl command {0}", new String(getSecureIngressPort));
+    try {
+      result = exec(new String(getSecureIngressPort), true);
+    } catch (Exception ex) {
+      logger.info("Exception in getSecureIngressPort() {0}", ex);
+      return 0;
+    }
+    logger.info("getSecureIngressPort: kubectl returned {0}", result.toString());
+    if (result.stdout() == null) {
+      return 0;
+    } else {
+      return new Integer(result.stdout());
+    }
+  }
+
+  /**
+   * Get the tcp ingress port of istio installation.
+   * @returns tcp ingress port for istio-ingressgateway
+   */
+  public static int getIstioTcpIngressPort() {
+    ExecResult result = null;
+    StringBuffer getTcpIngressPort = null;
+    getTcpIngressPort = new StringBuffer("kubectl -n istio-system get service istio-ingressgateway ");
+    getTcpIngressPort.append("-o jsonpath='{.spec.ports[?(@.name==\"tcp\")].nodePort}'");
+    logger.info("getTcpIngressPort: kubectl command {0}", new String(getTcpIngressPort));
+    try {
+      result = exec(new String(getTcpIngressPort), true);
+    } catch (Exception ex) {
+      logger.info("Exception in getTcpIngressPort() {0}", ex);
+      return 0;
+    }
+    logger.info("getTcpIngressPort: kubectl returned {0}", result.toString());
+    if (result.stdout() == null) {
+      return 0;
+    } else {
+      return new Integer(result.stdout());
+    }
+  }
+
+  /**
+   * Deploy the Http Istio Gateway and Istio Virtualservice.
+   * @param domainNamespace name of the domain namespace
+   * @param domainUid name of the domain identifier
+   * @param clusterServiceName  name of WebLogic cluster
+   * @param adminServiceName name of the admin service
+   * @returns true if deployment is success otherwise false
```

Comment:
```suggestion
   * @return true if deployment is success otherwise false
```

Reply:
done.  Shouldn't t be results in a checksyle error.